tests/test_utils.py

```diff
@@ -210,3 +211,17 @@ def test_wordfilter_multiple_filters_in_string() -> None:
     assert blacklisted_result.badword_indexes[0].index == 6
     assert blacklisted_result.badword_indexes[1].badword == "you"
     assert blacklisted_result.badword_indexes[1].index == 14
+
+
+# Email tests
+# =====================================================
+@pytest.mark.skip("Not running as it sends emails")
```

Comment:
Can you replace this with a real test? Appreciate there's not much in it, but still